.github/workflows/cicd-deploy-to-prod.yml

```diff
@@ -18,6 +18,7 @@ env:
   ADMIN_GITHUB_TOKEN: ${{ secrets.PROD_ADMIN_GITHUB_TOKEN }}
   FLASK_APP_SECRET: ${{ secrets.PROD_FLASK_APP_SECRET }}
   FLASK_CONFIGURATION: "production"
+  SEND_EMAIL_INVITES: "True"
```

Comment:
I'd recommend leaving this as false until we've run through the security assessment with the MvP Journey - never know what else we might find 🔒 👀

Reply:
We can temporarily turn it on in Dev to test, if needed 🧪

Reply:
Agreed, changed